core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala

```diff
@@ -34,11 +34,21 @@ private[ui] class PoolTable(pools: Map[Schedulable, PoolData], parent: StagesTab
     <table class="table table-bordered table-striped table-condensed sortable table-fixed">
       <thead>
         <th>Pool Name</th>
-        <th>Minimum Share</th>
-        <th>Pool Weight</th>
+        <th>
+          <span data-toggle="tooltip" data-placement="top" title="Pools can be set a minimum share
+           of CPU cores to allocate">Minimum Share</span>
+        </th>
+        <th>
+          <span title="This controls the pool's share of the cluster relative to other pools"
+           data-toggle="tooltip"  data-placement="top">Pool Weight</span>
+        </th>
         <th>Active Stages</th>
         <th>Running Tasks</th>
-        <th>SchedulingMode</th>
+        <th>
+          <span title="Mode to control whether jobs within the pool queue up behind each other
```

Comment:
Not sure if we can meaningfully elaborate the mode here, maybe leave it.
Remember that any real docs would exist in the documentation, not here

Reply:
ok i will remove the tooltips for mode.

Reply:
@srowen,  this document i referred to construct tooptips https://spark.apache.org/docs/3.0.0-preview/job-scheduling.html , please check "Configuring Pool Properties" section.